### ISSUE TITLE
the generated images have a yellowish tone

### ISSUE THREAD

**Issue by xinli2008** [kohya-ss/sd-scripts#1280]
Hello, I'm using this repository for LoRa training. I currently have a dataset of 700 samples. After training for 10 epochs, I tested my model and found that the generated images have a yellowish tone. I'm wondering why this is happening?

here is my config:

pretrained_model_name_or_path = "/mnt/sd-scripts-v2/pretrained_models/RealVisXL_V3.0.safetensors"
dataset_config = "/mnt/sd-scripts-v2/config/dataset_config.toml"
sample_prompts = "/mnt/sd-scripts-v2/config/prompt.txt"
output_dir = "/mnt/sd-scripts-v2/output_model/20240418/Asian-RealVisXL_llavacaption-32dimdim-16alpha-400data-float16-1convdim-1alpha-lr7e5-repeat10-noise_offset0-clipskip2"
logging_dir = "/mnt/sd-scripts-v2/logs"
output_name = "asian" 
train_batch_size = 12 
network_dim = 32 
network_alpha = 16 
conv_dim = 1 
conv_alpha = 1 
max_train_epochs = 60
save_every_n_epochs = 1
sample_every_n_epochs = 1
optimizer_type = "adaFactor"
lr_scheduler_num_cycles = 30
lr_scheduler = "cosine_with_restarts"
lr_warmup_steps = 0 
network_train_unet_only = false 
learning_rate = 0.00007 
unet_lr = 0.00007 
text_encoder_lr = 0.000008 
caption_extension = ".txt"
resolution = "1024,1024"
save_precision = "fp16"
mixed_precision = "fp16"
sample_sampler = "euler_a"
network_module = "networks.lora"
max_token_length = 225
bucket_reso_steps = 64
cache_latents = true
enable_bucket = true
bucket_no_upscale = true
xformers = true
persistent_data_loader_workers = true
gradient_checkpointing = true
cache_text_encoder_outputs = false 
no_half_vae = true 
max_data_loader_n_workers = 64 
save_model_as = "safetensors"
seed = 12345
noise_offset = 0
min_bucket_reso = 256
max_bucket_reso = 2048
scale_weight_norms = 1
clip_skip = 2

---- 
Is it the reason of noise offset ?

**Comment by xinli2008** [comment]
![image](https://github.com/kohya-ss/sd-scripts/assets/71311087/d39a2a7b-f988-4c24-ae48-2347e0e2d2d8)

**Comment by gesen2egee** [comment]
I've often been troubled recently by the colors in trained images not turning out as expected, possibly due to artifacts. 
It feels like this issue has become more frequent with the latest versions? 
I've tried switching datasets, adjusting noise, and changing SNR settings.
Or any recommended settings that might help avoid these?
